gns3/main_window.py

```diff
@@ -376,6 +378,9 @@ def _newProjectActionSlot(self):
             else:
                 self._createTemporaryProject()
 
+    def _IOUVMConverterActionSlot(self):
+        subprocess.Popen([shutil.which("gns3-iouvm-converter")])
```

Comment:
Also, should catch OSError and subprocess.SubprocessError exceptions like in https://github.com/GNS3/gns3-gui/blob/cd9bb16f79e8738fff34ff695730163b92c11170/gns3/serial_console.py#L59 for instance.